README.md

```diff
@@ -109,6 +109,8 @@ both for the command line and for other platforms:
   - [github.com/isacikgoz/tldr](https://github.com/isacikgoz/tldr):
     `go get github.com/isacikgoz/tldr`
     (or [platform binaries](https://github.com/isacikgoz/tldr/releases))
+  - [github.com/HardDie/myTldr](https://github.com/HardDie/myTldr):
+    `go get github.com/HardDie/myTldr`
```

Comment:
You can link your pre-built binaries:
```suggestion
    `go get github.com/HardDie/myTldr`
    (or [platform binaries](https://github.com/HardDie/myTldr/releases))
```

Reply:
Yeah, if you don't mind @HardDie that would be great!

Reply:
Job done